test/python/topology/test1.py

```diff
@@ -110,6 +110,37 @@ def test_TopologySourceAndSinkCallable(self):
       topo = Topology("test_TopologySourceAndSinkCallable")
       hw = topo.source(test_functions.SourceTuplesAppendIndex(["a", "b", "c", "d"]))
       hw.sink(test_functions.CheckTuples(["a0", "b1", "c2", "d3"]))
+    
```

Comment:
The logic being, if parallel were implemented as follows:

``` python
# Effectively do nothing.
def parallel(self, num_channels):
    return self
```

These tests would still pass.